controllers/users-controller.js

```diff
@@ -0,0 +1,80 @@
+import { Request, Response, NextFunction } from 'express';
+import bcrypt from 'bcryptjs';
+import createError from 'http-errors';
+import jwt from 'jsonwebtoken';
+import models from '../models';
+import { env } from '../helpers/utils';
+
+const generateToken = async payload => jwt.sign(payload, env('APP_KEY'));
+
+const { User } = models;
+
+/**
+ * Class representing users controller
+ *
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * Creates a new user resource
+   *
+   * @static
+   * @param {Request} request - Request object
+   * @param {Response} response - Response object
+   * @param {NextFunction} next - Express next function
+   * @returns {void}
+   *
+   * @memberof UsersController
+   */
+  static async index(request, response, next) {
```

Comment:
#### Nitpick
- Use a much more appropriate name for this method

Reply:
Working on resolving the failing builds. As regards the method name, will make modification.